host_vars/dalite.harvardx.harvard.edu/public.yml

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
But I mean, why should the `host_vars` be public, and why should it contain the default values? It doesn't make sense, especially since the `hosts` file is private. This should only contain host-specific configuration overrides, not the default configuration to share with every host. 

Can you move the default configuration to the role defaults (http://docs.ansible.com/ansible/playbooks_roles.html#role-default-variables) and move `host_vars` and its remaining host-specific variables to the private repo?